db/migrate/20240820123403_swap_big_int_notifications_id.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class SwapBigIntNotificationsId < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
```

Comment:
I don't think it is safe for us to disable transaction here.

Reply:
Yes, that is true and it fixes the other issue. I originally disabled it because I thought it was necessary for renaming columns.